bedrock/firefox/templates/firefox/whatsnew_42/base.html

```diff
@@ -0,0 +1,73 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
+{% add_lang_files "firefox/whatsnew_42" %}
+
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
+{% block page_og_title %}
+  {{_('Choose the independent browser')}}
+{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}
+  {{_('Firefox is non-profit, non-corporate, non-compromised. Choosing Firefox isn’t just choosing a browser. It’s a vote for personal freedom.')}}
+{% endblock %}
+
+{#- Override <meta property="og:url"> for social share -#}
+{% block page_og_url %}{{ url('firefox') }}{% endblock %}
+
+{% block optimizely %}
+  {% if waffle.switch('firefox-whatsnew-optimizely') %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{% block page_css %}{% endblock %}
+
+{% block page_title_prefix %}{% endblock %}
+{# L10n: the <br> tags below are for visual formatting only. #}
+{% block page_title %}{{ _('More protection. <br>The most privacy. <br>Only from Firefox.') }}{% endblock %}
+{% block page_title_suffix %}{% endblock %}
+{% block body_id %}firefox-whatsnew-42{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block site_header %}
+<header id="masthead">
+  <div id="tabzilla">
+    <a href="{{ url('mozorg.home') }}">Mozilla</a>
+  </div>
+  {% block up_to_date_message %}
+    <h2>{{ _('Congrats. You’re up to date!') }}</h2>
```

Comment:
This message is always shown regardless of the version of Fx I am using. Is that expected?